package/lite/etc/conf.d/log_paths/2/lp_dest_alts_global/plugin.jinja

```diff
@@ -1,64 +0,0 @@
-
```

Comment:
Using same config for `topics` routing that have SC4S. In general currently SC4S lite have support only 3-5 topics, but SC4S about 10 topics